pkg/errhelp/errors.go

```diff
@@ -64,6 +64,7 @@ const (
 	InvalidAddressPrefixFormat                     = "InvalidAddressPrefixFormat"
 	FeatureNotSupportedForEdition                  = "FeatureNotSupportedForEdition"
 	VirtualNetworkRuleBadRequest                   = "VirtualNetworkRuleBadRequest"
+	UnsupportedDiskCreateOption                    = "UnsupportedDiskCreateOption"
```

Comment:
These messages are used to categorize errors from the Azure SDK and aren't really meant to be error messages for the end user.